x86/SearchMacros.asm

```diff
@@ -87,7 +87,7 @@ Display	2, "Search(alpha=%i1, beta=%i2,	depth=%i8, cutNode=%i9)	called%n"
 		mov   dword[.bestValue], -VALUE_INFINITE
 	      movzx   r12d, byte[rbx-1*sizeof.State+State.ply]
 		add   r12d, 1
-		mov   byte[rbx+State.ply], r12l
```

Comment:
I am assuming Moha (or whoever wrote this section) meant "r12b" here, since this is the correct syntax to specify the lower byte of register r12.